src/_cffi_src/openssl/pkcs7.py

```diff
@@ -44,13 +77,17 @@
 PKCS7 *PKCS7_encrypt(Cryptography_STACK_OF_X509 *, BIO *,
                      const EVP_CIPHER *, int);
 int PKCS7_decrypt(PKCS7 *, EVP_PKEY *, X509 *, BIO *, int);
+
+BIO* PKCS7_dataInit (PKCS7 *, BIO *);
```

Comment:
Small style nits: the `*` should be next to `PKCS7`, not `BIO`, and no space after `Init`